distro.py

```diff
@@ -518,7 +518,7 @@ def __init__(self, f):
         self._f = f
 
     def __get__(self, obj, owner):
-        assert obj is not None, 'call {} on an instance'.format(self._fname)
+        assert obj is not None, 'call {0} on an instance'.format(self._fname)
```

Comment:
Genuinely curious, does 2.6 not support assuming format indexes?

Reply:
It doesn't. It's a 2.7 and up thing.

Reply:
Correct. See https://docs.python.org/2/library/string.html#format-string-syntax

> _Changed in version 2.7:_ The positional argument specifiers can be omitted, so `'{} {}'` is equivalent to `'{0} {1}'`.

Reply:
Thanks for verifying this!